src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinatorModel.swift

```diff
@@ -243,5 +243,32 @@ class ExposureSubmissionCoordinatorModel {
 			fatalError("Cannot set submission consent, no corona test type is set")
 		}
 	}
+	
+	func createDiaryTestEntry() {
```

Comment:
```suggestion
	func createCoronaTestEntryInContactDiary() {
```
At first time, I thought it is about to generate some test data 😅